samples/.NET/NUnit/Tests.cs

```diff
@@ -8,13 +8,11 @@ namespace PlaywrightTests;
 
 [Parallelizable(ParallelScope.Self)]
 [TestFixture]
-public class Tests : PageTest
+public class Tests : PageTestWithArtifact
```

Comment:
Sometimes user create their own playwright instance instead of using PageTest class as they want to play with context / page / browser which is easier if its self managed,, in that case will it work ?

Reply:
I am thinking of putting in similar fixtures for Browser, context as well.

Reply:
Added